src/org/loklak/harvester/TwitterScraper.java

```diff
@@ -416,9 +416,10 @@ public String toString() {
     
     public static class TwitterTweet extends MessageEntry implements Runnable {
 
-        public final Semaphore ready;
-        public UserEntry user;
-        public boolean writeToIndex, writeToBackend;
+        private final Semaphore ready;
+        private UserEntry user;
+        public boolean writeToIndex;
+        public boolean writeToBackend;

```

Comment:
Adding getter methods looks more proper. Otherwise it looks fine to me. Good job 👍